avm/ptn/azd/insights-dashboard/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,58 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-azd-insights-dashboard-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'aidmin'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+module dependencies 'dependencies.bicep' = {
+  name: '${uniqueString(deployment().name, resourceLocation)}-test-dependencies'
+  scope: resourceGroup
+  params: {
+    name: '${uniqueString(deployment().name, resourceLocation)}-test-log-analytics-${serviceShort}'
+  }
+}
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
```

```suggestion
resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
  name: resourceGroupName
  location: resourceLocation
}

module dependencies 'dependencies.bicep' = {
  name: '${uniqueString(deployment().name, resourceLocation)}-test-dependencies'
  scope: resourceGroup
  params: {
    name: '${uniqueString(deployment().name, resourceLocation)}-test-log-analytics-${serviceShort}'
  }
}
```

Just aligning it to the order of deployment